src/plugins/saved_objects/public/save_modal/saved_object_save_modal.test.tsx

```diff
@@ -31,6 +31,7 @@ describe('SavedObjectSaveModal', () => {
         title={'Saved Object title'}
         showCopyOnSave={false}
         objectType="visualization"
+        hideDescription={false}
```

Comment:
It’s just a nit but I’m just realizing there is a shiwCopyOnSave prop already - to be consistent can we make this a showDescription instead?

Reply:
Yes, of course.